cmd/user-list/main.go

```diff
@@ -17,8 +19,8 @@ var args struct {
 	config.ConfigArgs
 
 	Emails []string `help:"Optional list of users to limit the results to (leave empty for all users)." arg:"" optional:""`
-	Course string   `help:"Only include user enrolled in this course."`
 	Table  bool     `help:"Output data to stdout as a TSV." default:"false"`
+	Short  bool     `help:"Use short form output."`
```

Comment:
Now that we can see multiple CMDs, I think that `--short` doesn't work very well.
You should probably swap it to `--verbose`.
I'm thinking that we tell the user when they use `--verbose`,
the output will not be in any specific format (like JSON).
Then, you can always just output the full request and response (like we do in the Python interface).
And after you output the response, then you just do the output however you would do before.
This means that in some places there is kind of "double" output,
but that's okay since the user asked for verbose mode.